pkgs/development/python-modules/gntp/default.nix

```diff
@@ -0,0 +1,18 @@
+{ lib, buildPythonPackage, fetchPypi }:
+
+buildPythonPackage rec {
+  pname = "gntp";
+  version = "1.0.3";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1q6scs8lp84v0aph6b5c9jhv51rhq2vmzpdd38db92ybkq0g597l";
+  };
+
```

Comment:
There are no tests included. Please try to checkout from source and check if they have unit tests, and try to run them. Unit tests give a good indication that they package has a high degree of validity and correctness given the python package set.

If tests are not available, then please use `pythonImportsCheck` to import the most important modules. This isn't as good as unit tests, but will usually give a good indication of run-time errors. Please see [pythonImportsCheck documentation](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#using-pythonimportscheck) for more information.

```

----------------------------------------------------------------------
Ran 0 tests in 0.000s
```

Reply:
Indeed there are no testcase for this package. Thank you for taking the time to explain `pythonImportCheck`. Tell me if that's good enough

Reply:
there are tests: https://github.com/kfdm/gntp/tree/master/test

However, it looks like like you would need a growler instance running. I would just say something like:
```
  # requires a growler service to be running
  doCheck = false;
```

to denote why tests were excluded

Reply:
There might be manual tests but there are no testcase (iow not using a test framework).

Reply:
I have added the doCheck attribute